app/assets/stylesheets/application.sass.scss

```
@@ -10,6 +10,14 @@ dialog::backdrop {
   border-color: black;
 }
 
+.usa-table.usa-table--borderless.gray-header {
+  thead {
+    th {
+      background-color: #dfe1e2 !important;
```

this seems like it works without the `!important` rule, if you agree we can remove it
```suggestion
      background-color: #dfe1e2;
```